aicsimageio/readers/czi_reader.py

```diff
@@ -260,13 +289,16 @@ def _get_image_data(
         if read_dims is None:
             read_dims = {}
 
-        # Get current scene read dims
-        read_dims[CZI_SCENE_DIM_CHAR] = scene
-
         # Init czi
         with fs.open(path) as open_resource:
             czi = CziFile(open_resource)
 
+            # Get current scene read dims
+            adjusted_scene_index = CziReader._adjust_scene_index(
+                czi.get_dims_shape(), scene, czi.shape_is_consistent
+            )
+            read_dims[CZI_SCENE_DIM_CHAR] = adjusted_scene_index
+
```

Comment:
Thanks for flagging this.  The number of scenes reported by the file is 6 because czireader searches the metadata for scene names.  But the Dimensions reported by the file has "S":(1,2) which means less than 6 scene indices are actually valid!!  So this PR needs a bit more work to reconcile that.

Reply:
I guess the file is part of a split-scene CZI file, and this physical file only contains scene index 1 out of the 6 possible scenes.  But somehow the file metadata seems to include all 6 scene names.  I'm not sure if aicspylibczi can deal with split-scene czi.  I'm going to do a quick test on the parent file to see if it makes sense